.esh/commands/update-locales.py

```
@@ -495,32 +495,59 @@ def translate_value(value, source_lang, target_lang):
+	"placeholder",
+	"title",
+	"label",
+	"value",
+	"alt",
+	"aria-label",
+	"textContent",
+	"innerHTML",
+	"dataset",
+})
+
+# 同步过程中无法自动对齐的类型不匹配（脚本结束时 exit 1）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)